src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProject.cs

```diff
@@ -607,10 +607,11 @@ private async Task OnBatchScopeDisposedMaybeAsync(bool useAsync)
                     else
                     {
                         // TODO: find a cleaner way to fetch this
-                        var metadataReference = _projectSystemProjectFactory.Workspace.CurrentSolution.GetRequiredProject(Id).MetadataReferences.Cast<PortableExecutableReference>()
-                                                                                .Single(m => m.FilePath == path && m.Properties == properties);
+                        var metadataReference = _projectSystemProjectFactory.Workspace.CurrentSolution.GetRequiredProject(Id).MetadataReferences
+                            .Cast<PortableExecutableReference>()
+                            .Single(m => m.FilePath == path && m.Properties == properties);
 
-                        projectUpdateState = projectUpdateState.WithIncrementalReferenceRemoved(metadataReference);
+                        projectUpdateState = projectUpdateState.WithIncrementalMetadataReferenceRemoved(metadataReference);
```

Comment:
upcoming code is going to deal with AnalyzerReferences as well as MetadataReferences.  SO wanting the names to be clear ahead of that PR to cut down on churn.